tests/acceptance/05_processes/02_functions/findprocesses.cf

```diff
@@ -2,6 +2,9 @@
 #
 # Test findprocesses()
 #
+# Note: On HP-UX this depends on the
+# 00_hpux_ps_setup_serial.cf setup test having run first.
```

Comment:
I prefer not to duplicate the information. It's clearly explained why it's needed inside that test.